### DISCUSSION TITLE
Export data to PDF

### DISCUSSION THREAD

**Discussion by musumbidenis** [Power-Components/livewire-powergrid#648] [Q&A] [answered]
I need to generate a PDF from selected check-boxes. Has anyone achieved this to kindly help.

**Comment by luanfreitasdev** [comment, marked as answer]
I believe that in version 4 it will be possible.

See: https://github.com/Power-Components/livewire-powergrid/discussions/661

Once ready, we can generate excel using this too:
https://docs.laravel-excel.com/3.1/exports/export-formats.html

**Reply by musumbidenis** [reply]
That's great news.
Currently i've been able to create a bulk export button, passed my checkbox values to a route when the button is clicked. I got the checkbox values in the controller then did a query to fetch the data from db and finally exported my data to pdf using the dompdf package

**Comment by ashek1412** [comment]
is it possible in version 4 ?

**Reply by luanfreitasdev** [reply]
Yes, it is possible. However, you need to implement the feature outside of Powergrid.

https://github.com/Power-Components/livewire-powergrid/blob/4.x/resources/config/livewire-powergrid.php#L116

**Reply by ashek1412** [reply]
I want to generate pdf with visible columns. what u meant to implement a feature outside Powergrid?

**Reply by luanfreitasdev** [reply]
It is currently not supported natively, but it is possible
